### ISSUE TITLE
can't update Microsoft.CodeAnalysis.CSharp to 3.5

### ISSUE THREAD

**Issue by MagicAndre1981** [dotnet/roslyn#42493]
**Version Used**: 

[Microsoft.CodeAnalysis.CSharp 3.4.0](https://www.nuget.org/packages/Microsoft.CodeAnalysis.CSharp/3.4.0)

**Steps to Reproduce**:

1.  open nuget package ui
2.  try to update Microsoft.CodeAnalysis.CSharp from 3.4.0 to [3.5.0](https://www.nuget.org/packages/Microsoft.CodeAnalysis.CSharp/3.5.0)

**Expected Behavior**:

update works

**Actual Behavior**:

nuget fails to update because the stable version depends on a pre-release version of Microsoft.CodeAnalysis.Analyzers:

Microsoft.CodeAnalysis.CSharp 3.5.0 depends on Microsoft.CodeAnalysis.Common 3.5.0:

![image](https://user-images.githubusercontent.com/8274816/76831008-bf8dab00-6826-11ea-8771-fed005f57f47.png)

which depends on Microsoft.CodeAnalysis.Analyzers at least in version 3.0.0-beta2.20059.3:

![image](https://user-images.githubusercontent.com/8274816/76830757-3c6c5500-6826-11ea-8bfa-a8c1a6e52b53.png)

```
Attempting to gather dependency information for multiple packages with respect to project 'Foo.App.Android', targeting 'MonoAndroid,Version=v9.0'
Gathering dependency information took 3,57 min
Attempting to resolve dependencies for multiple packages.
Unable to resolve dependencies. 'Microsoft.CodeAnalysis.Analyzers 2.9.8' is not compatible with 'Microsoft.CodeAnalysis.Common 3.5.0 constraint: Microsoft.CodeAnalysis.Analyzers (>= 3.0.0-beta2.20059.3)'.
```

So release stable version 3.0 of **Microsoft.CodeAnalysis.Analyzers** to fix the nuget issue.

**Comment by MagicAndre1981** [comment]
I just figured out, that those packages were a dependency of my [used package HotReload to get XAML reloading before it was part of VS2019](https://github.com/AndreiMisiukevich/HotReload) and the author tried to implement code reloading, but this never worked in Visual Studio under Windows, only for him under MacOS for iOS/Android.

So I can remove them as he removed code reloading, but it would be still helpful to fix it for other users.

**Comment by JoeRobich** [comment]
@MagicAndre1981 Thanks for the report. We've now published version 3.0.0 of [Microsoft.CodeAnalysis.Analyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers/3.0.0) which has resolved this issue.

**Comment by MagicAndre1981** [comment]
> We've now published version 3.0.0 of [Microsoft.CodeAnalysis.Analyzers](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Analyzers/3.0.0) which has resolved this issue.

yes, update works now.
